### ISSUE TITLE
Update to meet accessibility standards

### ISSUE THREAD

**Issue by theamarks** [Seafood-Globalization-Lab/lab-manual#15]
Add details here

**Comment by jagephart** [comment]
From [accessibility checklist](https://www.washington.edu/accesstech/checklist/) s: 
- [x] Add descriptive alt text to all images
- [x] [Keyboard accessibility](https://www.washington.edu/accesstech/checklist/keyboard/) – Can all menus, links, buttons, and other controls be operated by keyboard, for users who are unable to use a mouse?
- [x] [Page regions](https://www.washington.edu/accesstech/checklist/regions/) – Are common regions of the web page properly identified in the code?
- [x] [ARIA](https://www.washington.edu/accesstech/checklist/aria/) – Do rich, dynamic, web interfaces include proper ARIA markup?
- [x] [Popups](https://www.washington.edu/accesstech/checklist/popups/) – Are pop-ups such as menus, dialogs, and tooltips accessible?
- [x] [Flashing and flickering](https://www.washington.edu/accesstech/checklist/flash-flicker/) – Have you avoided using content that flashes or flickers at a rate that might trigger seizures?
- [x] [Auto-updating content](https://www.washington.edu/accesstech/checklist/dynamic-content/) – Do features that update automatically (e.g., carousels, animated background graphics) have mechanisms that enable users to pause or avoid these features?
- [x] [Enlarged text](https://www.washington.edu/accesstech/checklist/enlarged-text/) – Does the content scale well when text is enlarged?
- [x] [Mobile devices](https://www.washington.edu/accesstech/checklist/mobile/) – Is content accessible on mobile devices?
- [x] [Predictability](https://www.washington.edu/accesstech/checklist/predictability/) – Have you avoided links, controls, or form fields that automatically trigger a change in context?
- [x] [Finding content](https://www.washington.edu/accesstech/checklist/finding/) – Does the website have consistent navigation, including two or more ways of finding content?
- [x] [Code validation](https://www.washington.edu/accesstech/checklist/validation/) – Is the web page coded using valid HTML?

**Comment by jagephart** [comment]
✅ page regions specified with headers 
✅ no relevant dynamic web interfaces, pop ups, flickering, auto updated text, etc.
✅ checked mobile accessibility
✅ html code is valid
